extensions/linz/README.md

```diff
@@ -3,7 +3,7 @@
 - **Title:** LINZ Top Level Extension
 - **Identifier:**
   <https://stac.linz.govt.nz/_STAC_VERSION_/linz/schema.json>
-- **Field Name Prefix:** linz
```

Comment:
I think this should stay with the word 'field'.
See conventions here: https://github.com/stac-extensions/datacube#datacube-extension-specification
and here https://github.com/stac-extensions/version